src/mmds/src/data_store.rs

```diff
@@ -16,12 +17,18 @@ pub struct Mmds {
 }
 
 /// MMDS version.
-#[derive(Clone, Copy, Debug, PartialEq)]
+#[derive(Clone, Copy, Debug, Deserialize, PartialEq, Serialize)]
 pub enum MmdsVersion {
     V1,
     V2,
 }
 
+impl Default for MmdsVersion {
+    fn default() -> MmdsVersion {
+        MmdsVersion::V1
```

Comment:
shouldn't V2 be the default version?

Reply:
See https://github.com/firecracker-microvm/firecracker/pull/2799#discussion_r773863628